README.md

```diff
@@ -145,7 +145,7 @@ server.statsd.set('your.set', 200);
 
 ## Version Compatibility
 
-### Currently compatible with: Hapi 16.x.x (Node v4)
+### Currently compatible with: Hapi 20.x.x
```

Comment:
This should probably be updated. 🤔 I see the `20.x.x` is a change that came from the original repo's `trunk` but your PR changes this all the way to `21`.